ceph_installer/tests/controllers/test_osd.py

```diff
@@ -76,6 +76,18 @@ def check(args, kwargs):
         result = session.app.post_json("/api/osd/configure/", params=data)
         assert result.status_int == 200
 
+    def test_configure_journal_devices(self, session, monkeypatch):
+        def check(args, kwargs):
+            extra_vars = kwargs["extra_vars"]
+            assert "journal_devices" not in extra_vars
```

Comment:
these asserts (given the previous comment) would then become tests on their own and they would not need a request at all